internal/store/compare.go

```diff
@@ -45,6 +51,11 @@ func Compare(srcDSN, dstDSN string, opts CompareOptions) ([]string, error) {
 				delete(srcTables, k)
 			}
 		}
+		for e := range incl {
+			if !strings.Contains(k, strings.ToLower(e)) {
```

Comment:
From the flag documentation, it looked like we should be doing an exact match? In that case, we could probably build up a map and check for this faster, rather than having 2 loops. But, not a blocker.